sky/utils/kubernetes/gpu_labeler.py

```diff
@@ -30,6 +30,20 @@ def prerequisite_check() -> Tuple[bool, str]:
     return prereq_ok, reason
 
 
+def check_nvidia_runtime_class() -> bool:
+    """Checks if the 'nvidia' RuntimeClass exists in the cluster"""
+    config.load_kube_config()
+    node_api = client.NodeV1Api()
+
+    # Fetch the list of available RuntimeClasses
+    runtime_classes = node_api.list_runtime_class()
+
+    # Check if 'nvidia' RuntimeClass exists
+    nvidia_exists = any(
+        rc.metadata.name == 'nvidia' for rc in runtime_classes.items)
+    return nvidia_exists
```

Comment:
Should we just use the functino in `sky.provision.kubernetes.instance` instead?

Reply:
good idea- done!